bodhi/server/models.py

```diff
@@ -2571,7 +2576,7 @@ def requested_tag(self):
             # release to the Release.dist-tag
             if self.release.state is ReleaseState.pending:
                 tag = self.release.dist_tag
-        elif self.request is UpdateRequest.testing:
+        elif self.request is UpdateRequest.testing or UpdateRequest.batched:
```

Comment:
This one also has the problem - the ```or UpdateRequest.batched``` will become ```or True```, which will make this if statement always be ```True```. You can use the ```in``` trick I described above, or repeat the ```self.request is``` on the second clause.